CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Enh #274: Refactor for compatibility with `yiisoft/db` package (@Tigrov)
 - Bug #277: Fix when there is a namespace but the directory does not exist (@Tigrov)
+- Enh #279: Use `ColumnBuilder` class to create table column definitions (@Tigrov)
```

Comment:
```suggestion
- Chg #279: Use `ColumnBuilder` class to create table column definitions (@Tigrov)
```

Seems, this is BC-break changes

Reply:
> Seems, this is BC-break changes

The PR extends API of the final `MigrationBuilder` class.
I don't think this will break any existing code.